upgrades/upgrade_system.gd

```diff
@@ -47,8 +47,11 @@ func attempt_upgrade(upgrade: Upgrade) -> void:
 	_purse.spend_coins(cost)
 	upgrade.level_increased.emit()
 
+	SignalBus.upgrade_completed.emit()
+
 
 func _handle_register_upgrade(upgrade: Upgrade) -> void:
+	print(upgrade.name)
```

Comment:
We'll probably want to either remove this or add more information as to not get it lost.

Reply:
yeah that got left in there while tracing some issues